### PR TITLE
✨ Add availabilty to study_file

### PR DESCRIPTION
**Pull request by parimalak**

### CODE REVIEW

**Review by znatty22 — APPROVED** [review]
💯

**Comment by dankolbman** [comment]
Do we want to define the values of `availability`?
Maybe just `available` or `cold storage`?

**Comment by znatty22** [comment]
I think like everything else, eventually this field will become an ENUM. We could define those values informally now and put it in the docstring tho

File: dataservice/api/study_file/models.py [hunk 1]
```diff
@@ -16,7 +16,8 @@ class StudyFile(db.Model, Base, IndexdFile):
     :param urls: Locations of file
     :param hashes: A dict keyed by hash type containing hashes of the file
     :param _metadata: A dict with any additional information
-    :param controlled_access: whether or not the file is controlled access
+    :param availability: Indicates whether a file is available for immediate
```
Comment (parimalak) [inline dataservice/api/study_file/models.py:19]: its in doc string

Comment (dankolbman) [reply dataservice/api/study_file/models.py:19]: yeah, I see that. Just wondering if we need to define these enums.

Comment (znatty22) [reply dataservice/api/study_file/models.py:19]: Will be figured out in #48 and implemented in #274

**Review by dankolbman — APPROVED** [review]

**Pull request merged**